pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1000,6 +1000,14 @@ def test_single_element_ix_dont_upcast(self, float_frame):
         result = df.loc[[0], "b"]
         tm.assert_series_equal(result, expected)
 
+    def test_iloc_callable_tuple_return_value(self):
+        df = DataFrame(np.random.randn(10, 4), index=range(0, 20, 2))
```

Comment:
Add `GH53769` at the top of the test function.

Reply:
Do you mean the link to this PR, or rather the link to the issue report (53533)?

Reply:
Done.